package.json

```diff
@@ -38,30 +38,34 @@
     "generate-docs": "api-documenter markdown --input-folder temp --output-folder docs && rm -rf temp",
     "generate-notices": "generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES --overwrite",
     "prepublishOnly": "npm run build",
-    "test": "jest --coverage",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test:unit": "jest --coverage --coverageDirectory=coverage/unit",
+    "test:visual": "./tests/scripts/visual-coverage.sh",
+    "test": "./tests/scripts/combined-coverage.sh"
   },
   "devDependencies": {
     "@babel/core": "^7.17.5",
     "@babel/preset-env": "^7.14.7",
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.14.5",
-    "@percy/cli": "^1.0.7",
-    "@percy/storybook": "^4.2.0",
-    "@storybook/addon-a11y": "^6.4.20",
-    "@storybook/addon-actions": "^6.4.19",
-    "@storybook/addon-essentials": "^6.4.19",
-    "@storybook/addon-interactions": "^6.4.19",
-    "@storybook/addon-links": "^6.4.19",
+    "@percy/cli": "^1.7.2",
```

Comment:
can these versions be upgraded like in my [pr](https://github.com/yext/search-ui-react/pull/274/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R51)? hopefully that way we can avoid a lot of merge conflicts in the package-lock

Reply:
sure